packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -76,8 +81,17 @@ export class RunningSummarizer implements IDisposable {
 			runtime,
 		);
 
+		// Before doing any heuristics or proceeding with its refreshing, lets refresh based on latest Summary ack if already available.
```

Comment:
Can you please add a little more detail why this is done. Basically, if there are any acks that were received while this summarizer catches up, we want to refresh state from it before attempting a summary?